### DISCUSSION TITLE
Feature Request - Floating Window Mode for Enhanced Multitasking

### DISCUSSION THREAD

**Discussion by sheng-di** [plateaukao/einkbro#355] [Ideas]
Dear EinkBro Development Team,

Firstly, I would like to express my appreciation for the work you've done on the EinkBro project. The app's focus on minimizing unnecessary UI transitions and animations, and its clear B&W icons, make it an excellent choice for Eink devices. The reading experience it provides is truly commendable.

I am writing to suggest a feature that I believe would greatly enhance the usability of the EinkBro browser - a Floating Window Mode. This mode would allow users to keep the browser floating on top of other apps, thereby enabling quick web searches or browsing while using other applications.

The Floating Window Mode could provide multiple benefits:

Efficiency: It would allow users to quickly reference or search for information on the web without needing to switch apps, thus improving multitasking and productivity.

Convenience: Users could keep relevant information visible while working in another app. This could be especially useful for tasks that require referencing multiple sources of information.

Customizability: Users could have the freedom to adjust the size and position of the floating window to best suit their needs and preferences.

I understand that implementing this feature may pose certain challenges. However, I am confident that it would be a valuable addition to the EinkBro browser, enhancing its overall functionality and user experience.

Thank you for considering my suggestion. I look forward to seeing the continued development and evolution of the EinkBro project.

**Comment by plateaukao** [comment]
@sheng-di 
Any other browser that provides this feature so that I can reference from ?
